### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by discordianfish**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.3.0 / 2016-05-05
+* [IMPROVEMENT] Use common/version and common/log packages (#44)
+* [CHANGE] Drop _count suffix for loaded_mappings metric (#41)
```
Comment (grobie) [inline CHANGELOG.md]: Escape the underscore then the syntax highlighting won't be off.

Comment (grobie) [inline CHANGELOG.md]: And let's list breaking changes first.

**Comment by grobie** [comment]
:+1:

**Comment by sdurrheimer** [comment]
Same as node_exporter, should we add there is a version flag now ? Otherwise 👍

**Comment by SuperQ** [comment]
Merged the CircleCI fix, please rebase.

**Comment by discordianfish** [comment]
Rebased and changes CHANGELOG.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## 0.3.0 / 2016-05-05
+* [CHANGE] Drop _count suffix for loaded_mappings metric (#41)
```
Comment (grobie) [inline CHANGELOG.md]: nitpick again. Let's leave a newline between heading and list for consistency and readability. Use `<backtick>_count<backtick>` or `\_count` to not break syntax highlighting.

**Pull request merged**